src/test/java/org/folio/rest/impl/PieceApiTest.java

```diff
@@ -356,4 +362,95 @@ void deletePieceInternalErrorOnStorageTest() {
     logger.info("=== Test delete piece by id - internal error from storage 500 ===");
     verifyDeleteResponse(String.format(PIECES_ID_PATH, ID_FOR_INTERNAL_SERVER_ERROR), APPLICATION_JSON, 500);
   }
+
+  @Test
+  void deletePiecesByIdsTest2() {
+    logger.info("=== Test delete pieces by ids - item deleted ===");
+
+    String pieceId = UUID.randomUUID().toString();
+    List<String> ids = Arrays.asList(pieceId);
+    Boolean deleteHoldings = false;
+    JsonArray jsonArrary = new JsonArray(ids);
+    JsonObject jsonObject = new JsonObject()
+      .put("ids", jsonArrary);
+    //  .put("deleteHolding", deleteHoldings);
+
+    List <Piece> pieces = new ArrayList<>();
+
+
+    // Mock response setup for deletion
+    //MockServer.addMockEntry(PIECES_STORAGE, JsonObject.mapFrom(jsonObject).encode()); // Simplified mock data
```

Comment:
also remove commentds, if it is not helpful

Reply:
thank you for pointing it out ! will do